### PR TITLE
Compliance E2E tests

### PR DESCRIPTION
**Pull request by bendiggle**
Added e2e tests for compliance to ensure the pages follow the correct path.
Renamed pages directory to pageObjects
Edited current tests so that each page has its own test feature. E.G. All tests for mrn-date page is in one scenario instead of several.

### CODE REVIEW

File: test/e2e/codecept.conf.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ exports.config = {
         }
     },
     'include': {
-        'I': './pages/steps.js'
+        'I': './pageObjects/steps.js'
```
Comment (paulgain) [inline test/e2e/codecept.conf.js]: For consistency can we rename this directory to page-objects.

File: test/e2e/pageObjects/compliance/checkMrn.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    I.seeInCurrentUrl(urls.compliance.checkMRNDate);
+
+}
+
+function goToCorrectPageAfterCheckMRN(value, url) {
+
+    const I = this;
+
+    I.checkOption(`#CheckMRN_checkedMRN-${value}`);
+    I.click('input[type="submit"]');
```
Comment (paulgain) [inline test/e2e/pageObjects/compliance/checkMrn.js]: I much prefer to use I.click('Continue'); as it reads a lot easier. It there a good reason to change it?

Comment (bendiggle) [reply test/e2e/pageObjects/compliance/checkMrn.js]: Not sure why I didn't put Continue! Will change it now

File: test/e2e/pageObjects/compliance/checkMrn.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+const urls = require('urls');
+
+function goToCheckMrnPage(mrnDate) {
+
+    const I = this;
+
+    I.fillField('#MRNDate_day', mrnDate.date());
+    I.fillField('#MRNDate_month', mrnDate.month() + 1);
+    I.fillField('MRNDate_year', mrnDate.year());
```
Comment (paulgain) [inline test/e2e/pageObjects/compliance/checkMrn.js]: For consistency please include the `id`

**Pull request merged**